Cargo.toml

```diff
@@ -186,8 +187,11 @@ easy-ext = "0.2"
 ed25519-dalek = { version = "2.1.0", default-features = false }
 enum-map = "2.1.0"
 enumset = "1.0"
+env_logger = "0.11.5"
 ethabi = "18"
 expect-test = "1.3.0"
+# Renaming since currently the name of an unpublished workspace member collides with `near-jsonrpc-client`. Once #12908 is resolved, the crate here should be used with its original name.
+ext-near-jsonrpc-client = { version = "0.15.1", package = "near-jsonrpc-client" }
```

Comment:
By using a different name, it's possible to add `near-jsonrpc-client` from crates.io to the workspace dependencies. I think that's a slight improvement because it:

- reduces the number of dependencies that cannot be inherited from the workspace
- makes it more explicit that there are two different `near-jsonrpc-client` crates

Reply:
Looks like the amount of version conflicts make it not worth it, then using local `near-jsonrpc-client` is a better first step.